src/pages/Login.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+
+export class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isName: false,
+      isEmail: false,
+    };
+    this.checkEmail = this.checkEmail.bind(this);
+    this.checkName = this.checkName.bind(this);
+  }
+
+  checkEmail({ target }) {
```

Comment:
O que acha de colocar o value de dentro do target ao invés de desestruturar logo embaixo?
Aí ficaria dessa forma, por exemplo:
```suggestion
  checkEmail({ target: { value } }) {
```
E aí não seria necesário chamar o `value` na linha 15 😊